addon/addon/models/animator.ts

```diff
@@ -0,0 +1,161 @@
+import { assert } from '@ember/debug';
+import { Snapshot } from '../utils/measurement';
+import { AnimationDefinition } from './orchestration';
+import Sprite, { SpriteType } from './sprite';
+
+export interface IContext {
+  id: string | undefined;
+  element: Element; // TODO can we change this to HTMLElement
```

Comment:
We might but the reason this was probably Element could be that there is also SVGElement which we want to support as well. I'm not 100% sure if inlined SVGs fully fall under HTMLElement or are even then SVGElement. This is something we can check though.

Reply:
Now that I think of this I'm not even sure if you added this comment? 😂 did this come from a previous files? It looks familiar.